Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -106,10 +108,25 @@ public static MRTKOculusConfig Instance
         [Tooltip("Custom hand material to use for hand tracking hand mesh.")]
         private Material customHandMaterial = null;
 
+        public System.Action onCustomHandMaterialUpdate;
```

Comment:
public variables have uppercase to start -> OnCustomHandMaterialUpdate

Reply:
Made the change.